src/components/ModalFooterTask/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react"
+import { Task } from "../../types/Task"
+import { DatePicker } from "@material-ui/pickers"
+import { ChipDate } from "../ChipDate"
+import MaterialIcon from "@material/react-material-icon"
+import { Button } from "../Button"
+
+interface Props {
+  activeDate: Task["date"]
+  changeActiveDate: Function
+  date?: Task["date"]
+  toggle: () => void
+}
+
+export function ModalFooterTask({
+  activeDate,
+  changeActiveDate,
+  date,
+  toggle,
+}: Props) {
+  const [displayCal, setDisplayCal] = useState(false)
+  return (
+    <>
```

Comment:
⚒ This would make it easier to tell what's what when inspecting the DOM.
```suggestion
    <div className="ModalFooterTask">
```